src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsKernel.hpp

```diff
@@ -207,15 +214,22 @@ class Multiphase :
     /// Stack storage for the element local nodal incremental displacement
     real64 uhat_local[numNodesPerElem][numDofPerTrialSupportPoint];
 
-    real64 localFlowResidual[numMaxComponents];
-    real64 localDispFlowJacobian[numDispDofPerElem][numMaxComponents + 1];
-    real64 localFlowDispJacobian[numMaxComponents][numDispDofPerElem];
-    real64 localFlowFlowJacobian[numMaxComponents][numMaxComponents + 1];
-    real64 localVolBalanceResidual[1];
-    real64 localVolBalanceJacobian[1][numMaxComponents + 1];
+    real64 ( &localResidualMomentum )[numDispDofPerElem];
+    real64 ( &dLocalResidualMomentum_dDisplacement )[numDispDofPerElem][numDispDofPerElem];
```

Comment:
```suggestion
    real64 localResidualMomentum[numDispDofPerElem];
    real64 dLocalResidualMomentum_dDisplacement[numDispDofPerElem][numDispDofPerElem];
```
for consistency with the array declarations made just after (unless I am missing something)

Reply:
Actually, it is intended: `localResidualMomentum` and `dLocalResidualMomentum_dDisplacement` are references to `localResidual` and `localJacobian` defined in `ImplicitKernelBase` class ([here](https://github.com/GEOSX/GEOSX/blob/17f71922f0f4ea9d03db0b0e4080a42e18d106c3/src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsKernel.hpp#L189-L190)). Introducing mixed spaces will allow to overcome this non-consistent treatment.